From 5d822ddeb04a43af6de3af19d52cd00888c7f905 Mon Sep 17 00:00:00 2001
From: Lu Hui <luhui@sipeed.com>
Date: Tue, 20 Feb 2024 10:41:53 +0800
Subject: [PATCH 15/17] 
 boards/sg200x/sg2002_licheervnano_sd/sg2002_licheervnano_sd_defconfig:
 disable rndis script, enable acm script

---
 .../sg2002_licheervnano_sd/sg2002_licheervnano_sd_defconfig     | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/boards/sg200x/sg2002_licheervnano_sd/sg2002_licheervnano_sd_defconfig b/boards/sg200x/sg2002_licheervnano_sd/sg2002_licheervnano_sd_defconfig
index 16c36c9..f685cbe 100644
--- a/boards/sg200x/sg2002_licheervnano_sd/sg2002_licheervnano_sd_defconfig
+++ b/boards/sg200x/sg2002_licheervnano_sd/sg2002_licheervnano_sd_defconfig
@@ -28,4 +28,4 @@ CONFIG_TARGET_PACKAGE_BUSYBOX_SYSLOGD_SCRIPT=y
 CONFIG_TARGET_PACKAGE_GATORD=n
 CONFIG_TARGET_PACKAGE_NTP=y
 CONFIG_TARGET_PACKAGE_WIFI=y
-CONFIG_TARGET_PACKAGE_RNDIS_SCRIPT=y
+CONFIG_TARGET_PACKAGE_ACM_SCRIPT=y
-- 
2.40.1

